localisation/00_wc_random_artifact_names.csv

```diff
@@ -92,6 +92,19 @@ artifact_legendary_origin_8;[GetGrammaticalNumberThisThese] [This.GetName] [GetG
 artifact_legendary_origin_9;[GetGrammaticalNumberThisThese] [This.GetName] holds the secret to existence for those who can decipher [GetGrammaticalNumberItsTheir] intricate design;ce [This.GetName] d�tient le secret de l'existence pour ceux qui peuvent d�chiffrer sa conception complexe;;;;;;;;;;;;x
 artifact_legendary_origin_10;materials from what [GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberWasWere] crafted do not exist in this world;les mat�riaux � partir desquels ce [This.GetName] a �t� cr�� n'existent pas dans ce monde;;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
+artifact_looted_origin_1;Some individual was kind enough to "donate" [GetGrammaticalNumberThisThese] [This.GetName] to soldiers of [target_raiding_ruler.GetTitledFirstName] when they arrived in [target_looted_holding.GetName]. Locals did not hear about that person ever since.;;;;;;;;;;;;;x
+artifact_looted_origin_2;Before [GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberWasWere] become a war trophy of [target_raiding_ruler.GetTitledFirstName], [GetGrammaticalNumberItThey] [GetGrammaticalNumberWasWere] belonging to a family living in [target_looted_holding.GetName], that ceased to exist after the encounter with unidentified armed forces.;;;;;;;;;;;;;x
+artifact_looted_origin_3;By the right of might, [target_raiding_ruler.GetTitledFirstName] become the lawful owner of [GetGrammaticalNumberThisThese] [This.GetName] after [target_raiding_ruler.GetHerHis] army sacked [target_looted_holding.GetName].;;;;;;;;;;;;;x
+artifact_looted_origin_4;Once upon a time, in [target_looted_holding.GetName] lived a man who thought that resistance to evil by force is morally wrong. Unfortunately for him, troops of [target_raiding_ruler.GetTitledFirstName] did not care about that when they took [GetGrammaticalNumberThisThese] [This.GetName] from his dead body.;;;;;;;;;;;;;x
+artifact_looted_origin_5;A poor widow that lived in [target_looted_holding.GetName] did not sell [GetGrammaticalNumberThisThese] [This.GetName] in order to get some food only because [GetGrammaticalNumberItThey] [GetGrammaticalNumberWasWere] the last memento of her husband. When raiders of [target_raiding_ruler.GetTitledFirstName] come, they send that widow to her husband.;;;;;;;;;;;;;x
```

Comment:
Made a few changes
```suggestion
artifact_looted_origin_5;A poor widow that lived in [target_looted_holding.GetName] did not sell [GetGrammaticalNumberThisThese] [This.GetName] in order to get some food only because [GetGrammaticalNumberItThey] [GetGrammaticalNumberWasWere] the last memento of her husband. When raiders of [target_raiding_ruler.GetTitledFirstName] arrived, they sent that widow to her husband.;;;;;;;;;;;;;x
```